From 981e6c04139cf2c9c90f8062ffbecf335325b67f Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Fri, 28 Jun 2019 15:43:54 +0530
Subject: [PATCH 1/3] board: fsl: ls1043avnprdb: modify board name to
 MPC-LS-VNP-RDB

cosmetic change: Modify board name to MPC-LS-VNP-RDB.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 arch/arm/dts/fsl-ls1043a-dcm.dts        | 2 +-
 board/freescale/ls1043ardb/ls1043ardb.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/fsl-ls1043a-dcm.dts b/arch/arm/dts/fsl-ls1043a-dcm.dts
index 75d103ee20..6d42380164 100644
--- a/arch/arm/dts/fsl-ls1043a-dcm.dts
+++ b/arch/arm/dts/fsl-ls1043a-dcm.dts
@@ -13,7 +13,7 @@
 /include/ "fsl-ls1043a.dtsi"
 
 / {
-	model = "LS1043AVNPRDB Board";
+	model = "MPC-LS-VNP-RDB Board";
 
         aliases {
 		spi0 = &qspi;
diff --git a/board/freescale/ls1043ardb/ls1043ardb.c b/board/freescale/ls1043ardb/ls1043ardb.c
index c0c92a15ad..3a0d31cf04 100755
--- a/board/freescale/ls1043ardb/ls1043ardb.c
+++ b/board/freescale/ls1043ardb/ls1043ardb.c
@@ -87,7 +87,7 @@ int checkboard(void)
 	printf("SD1_CLK1 = %s, SD1_CLK2 = %s\n", freq[sd1refclk_sel], freq[0]);
 
 #elif defined (CONFIG_TARGET_LS1043AVNPRDB)
-	printf("Board: LS1043AVNPRDB ");
+	printf("Board: MPC-LS-VNP-RDB ");
 
 	int rev;
 	rev = get_board_version();
-- 
2.17.1

